cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/async/OperationFuture.java

```diff
@@ -19,10 +19,12 @@
   private final String _operation;
   private final OperationProgress _operationProgress;
   private volatile Thread _executionThread = null;
+  private long _finishTime;
```

Comment:
The unit of `finishTime` is not clear. Is it in milliseconds, nanoseconds, seconds, ...? Let's update accordingly -- e.g. `_finishTime` -> `_finishTimeNs`